bin/numba

```diff
@@ -22,8 +22,8 @@ def make_parser():
                         help='[Deprecated] Dump the control flow graph')
     parser.add_argument('--dump-ast', action="store_true",
                         help='[Deprecated] Dump the AST')
-    parser.add_argument('--fancy', action='store_true',
-                        help='Try to output fancy files (.dot or .html)')
+    parser.add_argument('--fancy', nargs=1, metavar='html_filename',
```

Comment:
About needing the nargs argument. AFAIK, argparse arguments expect a value by default (i.e., by default it uses `action='store'`).